packages/lib-react-components/src/ZooHeader/components/NarrowMenuNavListItem/NarrowMenuNavListItem.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { useTranslation } from 'react-i18next'
 
 import SpacedText from '../../../SpacedText'
 
-export default function NarrowMenuNavListItem ({ color, text }) {
+export default function NarrowMenuNavListItem ({ color, text, overrideLang }) {
+  const { i18n } = useTranslation()
   return (
     <SpacedText
       color={color}
-      weight='bold'
+      lang={overrideLang ? 'en' : i18n.language}
```

Comment:
Is this setting the navigation menu to English, even if the translated text is in another language?

Reply:
It is setting the label of the dropdown button who's text is username.